src/Hedwig/ClientApp/src/hooks/useApi.ts

```
@@ -73,6 +76,27 @@ export default function useApi<TData>(
 			})
 		) : null;
 
+  // Create error parsing function
+	const handleError = async (error: any) => {
+		let jsonResponse: Object | null = null;
+		try {
+			jsonResponse = JSON.parse(await error.text());
+		} catch {}
+
+		let apiError: ApiError | null = null;
+		if(jsonResponse) {
```

this means that if the error thrown is not json-parse-able, the error obj will remain null

since we have an API spec, the only valid errors returned from the API are the types defined in the generated client models, which come from Json.

right now the only non-json error bodies we get back are the empty 403s, which happen in dev and staging, and I think are best ignored.

However i'm not sure that totally ignoring non-json errors is the right move here. Maybe we want to throw an error?